### PR TITLE
Replace http_requests_total metric

### PR DESCRIPTION
**Pull request by jamtur01**

### CODE REVIEW

**Comment by jamtur01** [comment]
Fixes #1074

File: content/docs/introduction/first_steps.md [hunk 1]
```diff
@@ -86,24 +86,24 @@ tab.
 
 As you can gather from http://localhost:9090/metrics, one metric that
 Prometheus exports about itself is called
-`http_requests_total` (the total number of HTTP requests the Prometheus server has made). Go ahead and enter this into the expression console:
+`promhttp_metric_handler_requests_total` (the total number of scrapes the Prometheus server has made). Go ahead and enter this into the expression console:
```
Comment (brian-brazil) [inline content/docs/introduction/first_steps.md]: This is the number of /metrics requests Prometheus has served.

Comment (RichiH) [reply content/docs/introduction/first_steps.md]: @jamtur01 do you want to update? Else, can I steal it so we can merge? Thanks! :)

**Comment by RichiH** [comment]
@jamtur01 do you want to update? Else, can I steal it so we can merge? Thanks! :)

Same comment, just in main body of PR, not in a line-wise comment.

File: content/docs/introduction/first_steps.md (hunk 1 above)
Comment (jamtur01) [reply content/docs/introduction/first_steps.md]: Fixed! Sorry missed this.

**Comment by jamtur01** [comment]
This should now be ready to go.

File: content/docs/introduction/first_steps.md [hunk 2]
```diff
@@ -86,24 +86,24 @@ tab.
 
 As you can gather from http://localhost:9090/metrics, one metric that
 Prometheus exports about itself is called
-`http_requests_total` (the total number of HTTP requests the Prometheus server has made). Go ahead and enter this into the expression console:
+`promhttp_metric_handler_requests_total` (the total number of `/metrics` requests the Prometheus server has made). Go ahead and enter this into the expression console:
```
Comment (brian-brazil) [inline content/docs/introduction/first_steps.md]: served, not made

Comment (brian-brazil) [reply content/docs/introduction/first_steps.md]: You might want prometheus_http_request_duration_seconds_count instead

Comment (jamtur01) [reply content/docs/introduction/first_steps.md]: @brian-brazil Updated. Thanks.

**Comment by brian-brazil** [comment]
Thanks!

**Pull request merged**